### DISCUSSION TITLE
CSS attached to components

### DISCUSSION THREAD

**Discussion by Nicjus** [wingsuit-designsystem/wingsuit#207] [Q&A]
Hello,

I create a new component with attached CSS file.
In storybook, everything is ok.
The I copy paste the code from docs tab to a drupal twig template.
I run "yarn build:drupal"

The component is well on the drupal page, but not the CSS.
Even if the css file is well created in dist folder.

What is missing ?

**Comment by christianwiedemann** [comment]
If you use the component with UI Patterns you can add the css inside the wingsuit.yml. Check out the organisms/slider.
If you don't use it with UI Patterns you need to include it inside your theme libraries file.

**Reply by Nicjus** [reply]
Thanks. I understood, now.
